store/hooks/saga.js

```diff
@@ -0,0 +1,55 @@
+import { call, put, take, takeLatest } from 'redux-saga/effects';
+import { camelizeKeys } from 'humps';
+import { actions as hooksActions } from './slice';
+import api from '../../api/hooks';
+import { GET_HOOKS_REQUEST, GET_SLACK_ENTITIES_REQUEST, SET_HOOK_REQUEST } from '../types';
+
+// eslint-disable-next-line max-statements
+function *getHooksRequest({ payload }) {
+  yield put(hooksActions.start());
+  try {
+    const response = yield call(api.requestHooks, payload);
+    const { data } = camelizeKeys(response).data;
+    const inHooks = [];
+    const outHooks = [];
+    for (let i = 0; i < data.length; i++) {
```

Comment:
quizás es un poquito más claro usar directamente `data.forEach` en vez de un loop for corriente

Reply:
sí! , esto viene corregido desde otra branch.